codec/types/any.go

```diff
@@ -127,6 +140,12 @@ func (any *Any) ClearCachedValue() {
 	any.cachedValue = nil
 }
 
+// UnsafeSetCachedValue sets cached value to a given interface.
+// This method is for internal use only!
+func (any *Any) UnsafeSetCachedValue(x interface{}) {
```

Comment:
This function isn't actually used anywhere, is it still necessary?

Reply:
Ah, I forgot to remove it. It's related to our chat 1h ago. I've added `NewAnyWithCustomTypeURL` instead.
I will remove this method.